proto/kvrpcpb.proto

```diff
@@ -287,13 +317,27 @@ message RawScanRequest {
     Context context = 1;
     bytes start_key = 2;
     uint32 limit = 3;
+    bool key_only = 4;
 }
 
 message RawScanResponse {
     errorpb.Error region_error = 1;
     repeated KvPair kvs = 2;
 }
 
+message RawBatchScanRequest {
+    Context context = 1;
+    repeated bytes start_keys = 2; // start keys for each scanning range
```

Comment:
em, if you want to support end keys here, I prefer using `repeated KeyRange ranges`. 

If you just want to avoid the case that scanning key1 + limit exceeding the next key2, you can sort the start keys in the server at first like `[a, b, c,]`, then use `[a, b)` as the range for the first scan, `[b, c)` for the second, then go on.

Reply:
End key is not for limiting records past next start key. It actually meant I'd like to stop at this end key even if number of scanned records does not go beyond limit. I prefer defining a KeyRange as well, didn't know it would be appropriate since other messages used two distinct fields. Let me update it.